test/test_uintwide_t_spot_values.cpp

```diff
@@ -41,6 +41,60 @@ bool math::wide_integer::test_uintwide_t_spot_values()
 {
   bool result_is_ok = true;
 
+  {
+    // See also https://github.com/ckormanyos/wide-integer/issues/154
+
+    using local_uint64_type    = math::wide_integer::uint64_t;
+    using local_uint128_type   = math::wide_integer::uint128_t;
+    using local_uint512_type   = math::wide_integer::uint512_t;
+    using local_uint1024_type  = math::wide_integer::uint1024_t;
+    using local_uint2048_type  = math::wide_integer::uint2048_t;
+    using local_uint4096_type  = math::wide_integer::uint4096_t;
+    using local_uint8192_type  = math::wide_integer::uint8192_t;
+    using local_uint16384_type = math::wide_integer::uint16384_t;
+    using local_uint32768_type = math::wide_integer::uint32768_t;
+    using local_uint65536_type = math::wide_integer::uint65536_t;
+
+    using local_int64_type    = math::wide_integer::int64_t;
+    using local_int128_type   = math::wide_integer::int128_t;
+    using local_int512_type   = math::wide_integer::int512_t;
+    using local_int1024_type  = math::wide_integer::int1024_t;
+    using local_int2048_type  = math::wide_integer::int2048_t;
+    using local_int4096_type  = math::wide_integer::int4096_t;
+    using local_int8192_type  = math::wide_integer::int8192_t;
+    using local_int16384_type = math::wide_integer::int16384_t;
+    using local_int32768_type = math::wide_integer::int32768_t;
+    using local_int65536_type = math::wide_integer::int65536_t;
+
+    result_is_ok &=
+      (
+           ((std::numeric_limits<local_uint64_type   >::max)() != 0U)
+        && ((std::numeric_limits<local_uint128_type  >::max)() != 0U)
```

Comment:
I've noticed that because all of the results are concatenated together with `&&`, it makes it hard to tell what failed. You could write each of these as separate `static_assert` statements and then it would always be obvious what had broken and where to start looking for fixes.

For run-time checks, `assert` would make the test fail fast and loud whenever it encountered a problem. Figuring out which specific condition was `false` would then not be additional effort.

Reply:
Agreed. I will modify accordingly.

Also, although I've not had time for larger issues, I also still recall #131 --- not yet done, but not forgotten either.

Reply:
I forgot that issue! Sorry to repeat myself. :)

Reply:
Thx John. I'm sure we'll keep moving forward this year as I hope to get to the bulk of the planned refactoring...